crates/generator/src/types.rs

```diff
@@ -1,10 +1,41 @@
 use gw_common::H256;
-use gw_types::packed::{StartChallenge, StartChallengeWitness};
+use gw_types::{
+    bytes::Bytes,
+    packed::{self, StartChallenge, StartChallengeWitness},
+    prelude::*,
+};
 use std::{
     collections::HashMap,
     fmt::{self, Display},
 };
 
+#[derive(Debug, PartialEq, Clone, Eq, Default)]
+pub struct LogItem {
```

Comment:
Can we use `packed::LogItem` directly?

Reply:
Will be a bit inconsistent with other fields in `RunResult` since they are all native types.

Reply:
Actually, `RunResult` is the only struct that needs to be native because the fields are `HashMap` type. Other structs such as `TxReceipt` are already migrated into the Molecule version.

Reply:
Ok